homeworks/Natalia.Alifirovets_NataMimoza/0-intro/README.MD

```diff
@@ -0,0 +1,2 @@
+node v20.8.0
+npm 10.1.0
```

Comment:
Hey, @NataMimoza, please add path to your node.js - point 6 in the task https://github.com/MastersAcademy/fe-course-basic-2023/issues/3#issue-1931677593